djstripe/models.py

```diff
@@ -106,7 +124,7 @@ def __str__(self):
 @python_2_unicode_compatible
 class Event(StripeObject):
     stripe_api_name = "Event"
-
+    objects = models.Manager()
```

Comment:
I don't think I can take out the default objects definition.

1) tests dont run. lots of exceptions
2) docs seem to say that you need to declare a manager  in the sub class

https://docs.djangoproject.com/en/1.8/topics/db/managers/#s-custom-managers-and-model-inheritance